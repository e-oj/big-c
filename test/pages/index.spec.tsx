import Index from '@pages/index';
import { render } from '@test/utils';

jest.mock('@lib/hooks', () => require('@mocks/hooks'));

describe('Homepage', () => {
    test('renders correctly', () => {
        const { container } = render(<Index />);

        expect(container.firstChild).toMatchSnapshot();
    });
});
