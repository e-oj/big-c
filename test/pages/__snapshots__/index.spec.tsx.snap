// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders correctly 1`] = `
<div>
  Hello World
</div>
`;
